Bottom: 747d4554de2c9979ee7644ccff0d9921444125d5
Top:    2db432b1639e1a37dd172b1505a98de5d70f26b3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-10 23:07:44 -0500

Fix outline drawing to only apply on copper layers

This matches the old comments in the code, and fixes (removes) outline
drawing on the solder mask prints.

This partially reverts commit 294494e56695251e63c0da9f243d6f59447f1c9f
"Fix PS rect/outline logic", which changed the logic, and
commit d2458383a60b670ad1c42163ac24fcc63fcab457 where I changed
the comment to match the code.


---

diff --git a/src/hid/ps/ps.c b/src/hid/ps/ps.c
index 31eea9d..b40303b 100644
--- a/src/hid/ps/ps.c
+++ b/src/hid/ps/ps.c
@@ -996,6 +996,7 @@ ps_set_layer (const char *name, int group, int empty)
      we want to "print outlines", and we have an outline layer,
      print the outline layer on this layer also.  */
   if (global.outline &&
+      global.is_copper &&
       global.outline_layer != NULL &&
       global.outline_layer != PCB->Data->Layer+idx &&
       strcmp (name, "outline") != 0 &&
